modules/rostests/apitests/win32nt/ntdd/NtGdiDdQueryDirectDrawObject.c

```diff
@@ -600,9 +601,6 @@ START_TEST(NtGdiDdQueryDirectDrawObject)
     RTEST(puD3dBufferCallbacks != NULL);
     ASSERT(puD3dDriverData != NULL);
 
-    RTEST(puD3dTextureFormats != NULL);
```

Comment:
Why is this removed? Can this pointer be NULL from time to time even e.g. on Windows?

Reply:
Please check the code. And then tell me why it wouldn't be NULL.